app/views/users/new.html.erb

```diff
@@ -4,6 +4,10 @@
 <% end %>
 
 <div class="auth-container">
+  <% if @client_app_name %>
+    <p class="association-note mt-4"><%= t(".login_to_authorize_html", :client_app_name => @client_app_name) %></p>
```

Comment:
That translation name should probably be `signup_to_authorize_html` here? Though see later comment about the translations...